apps/judicial-system/backend/src/app/modules/case/case.controller.ts

```diff
@@ -231,6 +244,16 @@ export class CaseController {
       update.courtRecordSignatureDate = null
     }
 
+    if (appealState !== undefined) {
+      // TODO: What if user is neither prosecutor nor defender?
```

Comment:
I haven't completely thought this through yet. I feel like I'm unsure how defenders work here since they aren't the same types of users as prosecutors, judges etc. Can a prosecutor even get here since they don't have a rolesRules transition guard?

Reply:
I think the safest thing would be to not handle defenders here.